src/token/api/src/canister/icrc1_transfer.rs

```diff
@@ -94,6 +110,50 @@ mod tests {
         canister
     }
 
+    #[test]
+    fn minting_with_nonzero_fee() {
+        let (_ctx, canister) = test_context();
+
+        let minter = AccountInternal::new(canister.state.borrow_mut().stats.owner, None);
+        let to = Account::from(bob());
+
+        let transfer = TransferArgs {
+            from_subaccount: Some(minter.subaccount),
+            to,
+            amount: Tokens128::from(100),
+            fee: Some(1.into()),
+            memo: None,
+            created_at_time: None,
+        };
+
+        assert!(
+            !canister.icrc1_transfer(transfer).is_ok(),
```

Comment:
I wonder why you used `not is ok` instead of `is err` :)

Reply:
good catch, thanks 👍